app/assets/javascripts/application.js

```diff
@@ -13,3 +13,4 @@
 //= require jquery
 //= require jquery_ujs
 //= require bootstrap
+//= require registration
```

Comment:
If you require it here then it is going to be rendered to all your views. What we want to do - render it only in your users/new template and nowhere else.